### ISSUE TITLE
Unable to click on "export/import/clear ad" in vault

### ISSUE THREAD

**Issue by cqx931** [dhowe/AdNauseam#587]
This should be caused by the latest fix for text overlaps in vault: https://github.com/dhowe/AdNauseam/commit/0fa2ec1b41881499bebfe60df6b38d2b47a0ebc7#diff-822bf4e2083b7e94fcb5c71019cc8169R263
We need position:absolute for #actions.

**Comment by leoneckert** [comment]
@cqx931 the elements "actions" and "stats" used to be `position:absolute` and positioned with `left:xx; bottom:xx` and `bottom:xx; right:xx` respectively, but that caused them to overlap when adjusting the window too narrow. 
My idea was to put them both into a wrapper that is `position:absolute` and set both of them `position:relative` so they could slide below each other. I just checked and it seems like I didn't explicitly set them to `relative` - but now I did and they seem clickable AND width-responsive again. check if that works for you.
